### DISCUSSION TITLE
订阅功能可以自动安装依赖文件吗?

### DISCUSSION THREAD

**Discussion by InfernalAzazel** [whyour/qinglong#1981] [Q&A]
青龙 版本: 2.15.15

1. 我添加了 requirements.txt 运行仍然出现提示缺少xxx 模块
2. 可以自定义自动添加任务后的时间规则吗? 我发现自动添加任务貌似时间规则时随机

![2023-06-15_17-23](https://github.com/whyour/qinglong/assets/40333954/f6deac1f-de48-4afc-b364-f648ea2d0a3e)

**Comment by nashaofu** [comment]
同问一下，能否在订阅的时候就把依赖自动安装上，手动安装依赖太难受了。还有就是订阅脚本后，会把所有符合的文件都添加到任务里面，我希望能排除掉utils文件，在订阅时添加`blacklist`会导致utils都没有下载，从而导致任务不能执行 @whyour

**Reply by nashaofu** [reply]
另外还有就是订阅的任务能指定cronjob时间吗？不想订阅后还要到任务列表修改定时规则
